delivery_carrier_label_paazl/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     "name": "Carrier labels for paazl",
     "summary": "Print carrier labels for paazl",
-    "version": "12.0.1.1.2",
+    "version": "12.0.1.1.3",
```

Comment:
No need to change the version, since it will be automatically updated when merged.